flake.nix

```diff
@@ -25,6 +25,13 @@
 
     unionmount.url = "github:srid/unionmount";
     unionmount.flake = false;
+
+    emanote-template.url = "github:srid/emanote-template";
+    # Don't care about the output of emanote-template, only the source
+    emanote-template.flake = false;
+    # To filter emanote-template source
+    # Why nix-filter? Try filtering directories with `builtins.path`, then you will see why.
+    nix-filter.url = "github:numtide/nix-filter";
```

Comment:
Can we instead use https://www.tweag.io/blog/2023-11-28-file-sets/ from new nixpkgs?

Reply:
Will add it!